src/darwin/Framework/CHIP/zap-generated/MTRClusterConstants.h

```diff
@@ -136,6 +136,7 @@ typedef NS_ENUM(uint32_t, MTRClusterIDType) {
     MTRClusterApplianceEventsAndAlertID = 0x00000B02,
     MTRClusterApplianceStatisticsID = 0x00000B03,
     MTRClusterElectricalMeasurementID = 0x00000B04,
+    MTRClusterDiscoBallID = 0x00003456,
```

Comment:
What guarantees this will never be assigned to an actual cluster?  At the very least, shouldn't this cluster be under one of the test vendor ids?